### ISSUE TITLE
Error: Cannot read property 'pipe' of undefined (Pretty is undefined)

### ISSUE THREAD

**Issue by NAlexandrov** [hemerajs/hemera#203]
## Description
Example: Error thrown when Hemera start

## Expected Result
Hemera start successfully

## Actual Result
```
node_modules/nats-hemera/lib/index.js:225
        pretty.pipe(process.stdout)
               ^

TypeError: Cannot read property 'pipe' of undefined
```
## Context
- [x] nats-hemera (core)
- [ ] hemera-zipkin
- [ ] hemera-store
- [ ] hemera-stats
- [ ] hemera-rabbitmq
- [x] hemera-nsq
- [ ] hemera-elasticsearch
- [x] hemera-joi
- [ ] hemera-msgpack
- [ ] hemera-avro
- [ ] hemera-jwt-auth

## Your Environment
- NATS: 1.0.4
- Hemera: 5.4.0
- NodeJs: 8.11.1
- Operating System and version: OS X 10.13.4

**Comment by StarpTech** [comment]
Hi @NAlexandrov thanks for reporting I'm already working on this.

**Comment by StarpTech** [comment]
Is addressed in https://github.com/hemerajs/hemera/pull/205

**Comment by StarpTech** [comment]
Released with 5.4.1
